### PR TITLE
chore: fixing thingies

### PR DESCRIPTION
**Pull request by 1emu**

### CODE REVIEW

File: src/components/Common/Typography/List.tsx [hunk 1]
```diff
@@ -9,8 +10,8 @@ type ListProps = (React.HTMLAttributes<HTMLUListElement> | React.HTMLAttributes<
 
 export default function List(props: ListProps) {
```
Comment (andyesp) [inline src/components/Common/Typography/List.tsx:11]: You could also do `({ ordered, ... props })` and you don't have to use omit

File: src/components/Home/ScrollToHash.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react'
+import { useLocation } from 'react-router-dom'
+
+function ScrollToHash() {
```
Comment (andyesp) [inline src/components/Home/ScrollToHash.tsx:4]: Why isn't this a hook `useScrollToHash()`?

File: src/components/Common/Typography/Markdown.tsx [hunk 3]
```diff
@@ -125,7 +126,11 @@ function getComponents({ size, componentsClassNames }: MarkdownProps): Component
 
 function Markdown(props: MarkdownProps) {
   const components = useMemo(() => getComponents(props), [props])
-  return <ReactMarkdown {...props} components={components} remarkPlugins={plugins} />
+  return (
+    <ReactMarkdown {...props} components={components} remarkPlugins={plugins}>
+      {props.children}
```
[the remark this answers is not in the document]
Comment (1emu) [reply src/components/Common/Typography/Markdown.tsx:131]: Yeah, I made it explicit bc the inspection was complaining that it should have the children prop 🤷 and it should not be passed like the other props 🙃

File: src/components/Home/ScrollToHash.tsx (hunk 2 above)
Comment (1emu) [reply src/components/Home/ScrollToHash.tsx:4]: I can make it a hook instead

```typescript

function LayoutShell() {
  useScrollToHash()

  return (
      <IntlProvider defaultLocale="en" locale="en" messages={flattenMessages(en)}>
        <SnapshotStatus />
        <Layout>
          <Outlet />
        </Layout>
      </IntlProvider>
  )
}
```

Are there any pro/cons?

Comment (andyesp) [reply src/components/Home/ScrollToHash.tsx]: No, but it doesn't look like a component :P

**Review by andyesp — APPROVED** [review]

**Pull request merged**